OpenRA.Mods.Common/Traits/World/ActorSpawnManager.cs

```diff
@@ -13,20 +13,21 @@
 using System.Linq;
 using OpenRA.Primitives;
 using OpenRA.Traits;
+using OpenRA.Mods.Common;
 
 namespace OpenRA.Mods.Common.Traits
 {
 	[Desc("Controls the spawning of specified actor types. Attach this to the world actor.")]
-	public class ActorSpawnManagerInfo : ConditionalTraitInfo, Requires<MapCreepsInfo>
+	public class ActorSpawnManagerInfo : ConditionalTraitInfo, Requires<MapCreepsInfo>, IRulesetLoaded
```

Comment:
`ConditionalTraitInfo` already implements `IRulesetLoaded`, so this shouldn't be needed.

Reply:
Will fix later today.